example/src/release_tests/swipeableReanimation/index.tsx

```diff
@@ -48,55 +47,8 @@ function RightAction(prog: SharedValue<number>, drag: SharedValue<number>) {
   );
 }
 
-function LegacyLeftAction(prog: any, drag: any) {
-  prog.addListener((value: any) => {
-    console.log('[L] showLeftProgress:', value.value);
-  });
-  drag.addListener((value: any) => {
-    console.log('[L] appliedTranslation:', value.value);
-  });
-
-  const trans = Animated.subtract(drag, 50);
-
-  return (
-    <Animated.Text
-      style={[
-        styles.leftAction,
-        {
-          transform: [{ translateX: trans }],
-        },
-      ]}>
-      Text
-    </Animated.Text>
-  );
-}
-
-function LegacyRightAction(prog: any, drag: any) {
-  prog.addListener((value: any) => {
-    console.log('[L] showRightProgress:', value.value);
-  });
-  drag.addListener((value: any) => {
-    console.log('[L] appliedTranslation:', value.value);
-  });
-
-  const trans = Animated.add(drag, 50);
-
-  return (
-    <Animated.Text
-      style={[
-        styles.rightAction,
-        {
-          transform: [{ translateX: trans }],
-        },
-      ]}>
-      Text
-    </Animated.Text>
-  );
-}
-
 export default function Example() {
```

Comment:
If this is the yellowish looking one then I guess we can remove it entirely, no? It was supposed to compare both implementations visually and with one implementation gone it kinda lost its purpose.

Reply:
Makes sense, although right now it serves more as a test for the programatic swipeable controls, which encountered regressions several times since release.

<img width="354" alt="Screenshot 2024-12-06 at 14 35 22" src="https://github.com/user-attachments/assets/24fdb01c-a272-4299-93c8-b44153a5e246">

Reply:
done in [5599019](https://github.com/software-mansion/react-native-gesture-handler/pull/3230/commits/55990199c652625937884e15543377fafcc95df0)